shared/src/business/useCases/pdf/sanitizePdfInteractor.test.js

```diff
@@ -26,6 +30,9 @@ describe('sanitizePdfInteractor', () => {
     });
 
     applicationContext.getPersistenceGateway().deleteDocumentFromS3 = jest.fn();
+
+    applicationContext.getUseCaseHelpers().setPdfFields = jest.fn();
```

Comment:
Nit: setPdfFields and deleteDocumentFromS3 could be added to the `createTestApplicationContext` to follow our "mocking strategy" a little more closely

Reply:
Thanks for the heads up! Fixed it!